housekeeper/store/api/find.py

```diff
@@ -30,6 +32,14 @@ def _get_bundle_query(self) -> Query:
         """Return bundle query."""
         return self.Bundle.query
 
+    def _get_version_query(self) -> Query:
+        """Return version query."""
+        return self.Version.query
+
+    def _get_version_bundle_query(self) -> Query:
```

Comment:
Would be great to add simple unit test for these as they are orthogonal to the filters and might break when we retire alchy and/or update  sqlalchemy